lunatrace/bsl/hasura/migrations/lunatrace/1649717845749_create-users-table-use-as-auth-table/up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE public.users (
```

Comment:
there are a few other places we are currently referencing the "identities" table in the database, and I think those will need get updated to point at this table as well.  "creator_id" and so forth.  You should be able to find them by looking for `REFERENCES public.identities` or whatever. 

Also, we've got a lot of foreign keys here making this pretty much a join table at the moment, and this might be a good time to think about if there is anything else we might want to put on here that we are going to need later.  Things that we will want to keep up-to-date when webhooks fire, for instance. Im not sure, just asking.  Doesn't look like theres a webhook for the user changed their email or anything...